src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -85,9 +90,20 @@ export class FindAndReplacePlugin extends UIPlugin {
         this.clearSearch();
```

Comment:
It's possible that the reason for not clearing the search during UPDATE_CELL is because updating a cell's content does not affect the positions of the search matches. On the other hand, when adding or removing columns/rows, the search may become invalid because the column and row indexes of the cells have changed.

Reply:
Already resolved this issue.